client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -122,10 +125,9 @@ export function* createSite(
 			site_information: {
 				title: siteTitle,
 			},
-			site_creation_flow: isEnabled( 'gutenboarding/site-editor' )
-				? 'gutenboarding-site-editor'
-				: 'gutenboarding',
-			theme: `pub/${ selectedDesign?.theme || 'twentytwenty' }`,
+			site_creation_flow: 'gutenboarding',
+			enable_fse: shouldEnableFse,
+			theme: `pub/${ selectedDesign?.theme || defaultTheme }`,
```

Comment:
Odd that `defaultTheme` didn't need `pub/`? 🤔

Reply:
This should prefix both selected design and defaultTheme with `pub/`, right?

Reply:
Right.